.jshintrc

```diff
@@ -25,7 +25,7 @@
   "strict"        : false,    // true: Requires all functions run in ES5 Strict Mode
   "maxparams"     : 10,       // {int} Max number of formal params allowed per function
   "maxdepth"      : 4,        // {int} Max depth of nested blocks (within functions)
-  "maxstatements" : 32,       // {int} Max number statements per function
+  "maxstatements" : 64,       // {int} Max number statements per function
```

Comment:
Yup O:-) .. but this one should be a warning, not something that makes the tests fail.. as soon as I'm done making miq_bot lint JS too, I'd like to remove `jshint` and `jscs` from the SSUI tests and just have miq_bot run it..